kci_test

```diff
@@ -288,10 +286,8 @@ class cmd_generate(Command):
             print("Either --build-output or --install-path is required")
             return False
 
-        install = (
-            args.install_path
-            or kernelci.build.Step.get_install_path(None, args.build_output)
-        )
+        install = kernelci.build.Step.get_install_path(None, args.build_output,
+                                                       args.install_path)
```

Comment:
Maybe use keyword arguments:
```suggestion
        install = kernelci.build.Step.get_install_path(
            output_path=args.build_output, install_path=args.install_path
        )
```
I guess it's just a bit clearer and less prone to problems if the arguments are changing.